src/theme/Root.tsx

```diff
@@ -11,6 +13,12 @@ type TRootProps = {
   children: ReactNode;
 };
 
+if (siteConfig.customFields.trackJsToken) {
+  TrackJS.install({ token: siteConfig.customFields.trackJsToken });
```

Comment:
Do we need to type cast this to a `string` @markwylde-deriv, since test case failures are related to it? 🤔  
```suggestion
  TrackJS.install({ token: siteConfig.customFields.trackJsToken.toString() });
```

Reply:
Thanks Jim